theano/sandbox/cuda/blas.py

```diff
@@ -855,15 +855,18 @@ class BaseGpuCorrMM(GpuOp):
         or a pair of integers
     subsample
         Perform subsampling of the output (default: (1, 1)).
+    filter_dilation
+        Perform subsampling of the input, also known as dilation (default: (1, 1)).
     pad
         *deprecated*, now you should always use border_mode.
 
     """
 
     check_broadcast = False
-    __props__ = ('border_mode', 'subsample')
+    __props__ = ('border_mode', 'subsample', 'filter_dilation')
 
-    def __init__(self, border_mode="valid", subsample=(1, 1), pad=(0, 0)):
+    def __init__(self, border_mode="valid", subsample=(1, 1),
+                 filter_dilation=(1, 1), pad=(0, 0)):
```

Comment:
Even if `pad` is deprecated, you should probably add the new argument in the end to not break backwards compatibility for positional arguments. (What do the Theano devs say?)